core/soc/estimator.go

```diff
@@ -51,15 +54,42 @@ func (s *Estimator) Reset() {
 	s.capacity = float64(s.vehicle.Capacity()) * 1e3  // cache to simplify debugging
 	s.virtualCapacity = s.capacity / ChargeEfficiency // initial capacity taking efficiency into account
 	s.energyPerSocStep = s.virtualCapacity / 100
+	s.minChargePower = 1000  // default 1 kW
+	s.maxChargePower = 50000 // default 50 kW
```

Comment:
Idee:

    capacity * 3

Reply:
@andig auf was du hier technisch raus willst wäre die Laderate C.

Könnte man durchaus machen, aber auch da stellt sich direkt die Frage ob diese entweder hinreichend konstant über alle Akkus ist oder eine gängige, dem Anwender bekannte Größe für sein Fahrzeug ist.
Ich zumindest musste für meine Fahrzeuge schon länger überlegen obwohl ich für deinen Ansatz durchaus auch eine gewisse Sympathie habe.

Wie in diesem Artikel https://www.heise.de/autos/artikel/Die-C-Rate-Wie-Ladeleistung-und-Batteriegroesse-zusammenhaengen-4535987.html kurz dargestellt gibt es leider eine erhebliche Streuung in der Größenordnung 1-3 hinsichtlich den gängigen Laderaten.

Reply:
Ist denn in dem Punkt DC-laden mit AC-laden vergleichbar?